CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+- Committed banner for home page
```

Comment:
This can just be one line but worth clarifiying what `Committed` means. Something like 

- Committed (podcast series) archive page and banner